apps/deploy-web/src/context/WalletProvider/WalletProvider.tsx

```diff
@@ -320,6 +321,7 @@ export const WalletProvider: React.FC<{ children: React.ReactNode }> = ({ childr
         isCustodial: !isManaged,
         isWalletLoading: isLoading,
         isTrialing: isManaged && !!managedWallet?.isTrialing,
+        isOnboarding: !user?.userId,
```

Comment:
Sorry I just realized this doesn't make sense, we want to check if the user has an account, but not paid so trialing.
```suggestion
        isOnboarding: !!user?.userId && isManaged && !!managedWallet?.isTrialing,
```